.github/workflows/bump-channel-version/action.yml

```diff
@@ -34,7 +34,7 @@ runs:
             echo "ELEMENTOR_CHANNEL_PACKAGE_VERSION=${ELEMENTOR_CHANNEL_PACKAGE_VERSION}"
             echo "NEW_CHANNEL_VERSION=${NEW_CHANNEL_VERSION}"
 
-            # Override PACKAGE_VERSION value is used in the one click release workflow to set the release and file names
+            # An overriden PACKAGE_VERSION value is used in the one click release workflow to set the release and file names
             # So that they contain ther postfix and bump version number.
```

Comment:
```suggestion
            # An overriden PACKAGE_VERSION value is used in the one click release workflow to set the release and file names
            # So that they contain their postfix and bump version number.
            # Should be unified with the code in get-new-package-version.
```